doc/nrf/releases/release-notes-2.3.0.rst

```diff
@@ -0,0 +1,1034 @@
+
+  * nRF7002 Wi-Fi 6 companion IC and nRF7002 DK.
+  * Bluetooth® Low Energy: Period Advertising Sync Transfer.
+    This feature allows Periodic Advertising synchronization data to be transferred over an ACL connection, which gives power saving benefits for energy constraint devices.
+    This is a key feature for the `Auracast™`_ Assistant.
+  * nRF Cloud Location Services Wi-Fi® feature support in the :ref:`lib_location` library and demonstrated in :ref:`asset_tracker_v2`, :ref:`modem_shell_application`, and :ref:`lwm2m_client`.
+  * Writing modem trace to external flash for later retrieval.
+    The :ref:`modem_shell_application` sample now demonstrates how to store and upload modem trace to cloud.
+  * New :ref:`Serial LTE Modem (SLM) Shell <slm_shell_sample>` sample:
+    The sample demonstrates sending AT commands to the nRF9160 SiP from shell for nRF53 and nRF52 Series SoCs.
+  * New :ref:`mqtt_sample` sample supporting Wi-Fi together with cellular connectivity (replacing the Simple MQTT sample).
+
+* Added :ref:`experimental support <software_maturity>` for the following features:
+
+  * Periodic Advertisement with Responses (PAwR) - Advertiser.
+    This is a new feature introduced in Bluetooth v5.4 specification, enabling the bidirectional exchange of application data using connectionless communication.
+    Learn more in our `Bluetooth 5.4 DevZone blog`_.
+    The SoftDevice Controller continues to be Bluetooth v5.3 qualified.
+  * Bluetooth LE Audio: Public Broadcast Announcement (PBA) and bidirectional streams to and from two headsets.
+  * New :ref:`wifi_sr_coex_sample` sample.
+
+* Improved:
+
+  * The :ref:`matter_lock_sample` sample has been extended to support switching between Matter over Thread and Matter over Wi-Fi during application operation.
+  * The :ref:`wifi_shell_sample` sample by adding the Power Save feature.
+
+See :ref:`ncs_release_notes_230_changelog` for the complete list of changes.
+
+Sign up for the `nRF Connect SDK v2.3.0 webinar`_ to learn more about the new features.
+
+The official nRF Connect for VS Code extension has also been improved.
+See the `latest release notes for nRF Connect for Visual Studio Code`_ for more information.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.3.0**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.3.0`_.
+
+IDE and tool support
+********************
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the only officially supported IDE for |NCS| v2.3.0.
+SEGGER Embedded Studio Nordic Edition is no longer tested or expected to work with |NCS| v2.3.0.
```

Comment:
Do we need this line here still? This is the third release since the deprecation of SES, so we should be meeting the requirements to stop mentioning this in the docs (@tejlmand once said we need to announce deprecation and keep the info for 2 releases before we get rid of it).

Reply:
confirmed by Ingar that we can remove this sentence